common/app/common/dfp/DfpData.scala

```diff
@@ -148,13 +149,13 @@ case class GuTargeting(adUnitsIncluded: Seq[GuAdUnit],
                        geoTargetsExcluded: Seq[GeoTarget],
                        customTargetSets: Seq[CustomTargetSet]) {
 
-  val keyValues: Seq[String] = {
-    for {
-      targetSet <- customTargetSets
-      target <- targetSet.targets if target.isKeywordTag
-      targetValue <- target.values
-    } yield targetValue
-  }
+  // This is local pure helper function:
+  private val tagValues = ( ( customTargetSets: Seq[CustomTargetSet],
```

Comment:
Could you add a type to this?

Reply:
Actually, it is already implicit in the definition - sorry it wasn't clear because of the way GitHub has inserted the comments 👍

Reply:
Actually I was wondering about that. I would prefer to use the 4th form from https://docs.scala-lang.org/style/declarations.html#function-values but I couldn't get it to work.